app/code/Magento/CatalogUrlRewrite/Model/ResourceModel/Category/Product.php

```diff
@@ -19,12 +19,12 @@ class Product extends AbstractDb
     /**
      * Product/Category relation table name
      */
-    const TABLE_NAME = 'catalog_url_rewrite_product_category';
+    public const TABLE_NAME = 'catalog_url_rewrite_product_category';
 
     /**
      * Chunk for mass insert
      */
-    const CHUNK_SIZE = 100;
+    private const CHUNK_SIZE = 100;
```

Comment:
@amenk, @ihor-sviziev: isn't this a backwards incompatible change? The constant used to be public but is now private, this change will break for people who are relying on this constant, right?

Reply:
kind of true, shall we change it back?

Reply:
I think we should make it public back
```suggestion
    public const CHUNK_SIZE = 100;
```

Reply:
thanks :-)